capa/v26.0.0-alpha.1/release.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: release.giantswarm.io/v1alpha1
+kind: Release
+metadata:
+  name: v26.0.0-alpha.1
+spec:
+  apps:
+  - name: aws-ebs-csi-driver
+    version: 2.30.1
+    dependsOn:
+    - cloud-provider-aws
+  - name: aws-pod-identity-webhook
+    version: 1.14.2
+    dependsOn:
+    - cert-manager
+  - name: capi-node-labeler
+    version: 0.5.0
+  - name: cert-exporter
+    version: 2.9.0
+    dependsOn:
+    - kyverno
+  - name: cert-manager
+    version: 3.7.5
+    dependsOn:
+    - prometheus-operator-crd
+  - name: chart-operator-extensions
+    version: 1.1.2
+    dependsOn:
+    - prometheus-operator-crd
+  - name: cilium
+    version: 0.24.0
+  - name: cilium-crossplane-resources
+    version: 0.1.0
+  - name: cilium-servicemonitors
+    version: 0.1.2
+    dependsOn:
+    - prometheus-operator-crd
+  - name: cloud-provider-aws
```

Comment:
Does this refer to https://github.com/giantswarm/cluster-aws/blob/1c557003b5f5b3c1316ba17b6d5f88c24d55cbcd/helm/cluster-aws/templates/cloud-provider-aws-helmrelease.yaml#L19?

Reply:
yes